### PR TITLE
Initialise runbooks for Nuxt 2XX/5XX alarms

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Related to WordPress/openverse-infrastructure#606 by @dhruvkb

## Description

This PR adds runbooks for the alarms added in the above linked PR.

## Testing Instructions

Read the documentation pages from the preview site.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: documentation/meta/monitoring/runbooks/nuxt_5xx_above_threshold.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## Severity guide
+
+Confirm there is not an outage.
+
+Determine the source of the 5XX errors (this can be checked by observing paths
+in the Cloudflare logs).
+
+- If the API requests are returning 2XX responses, the severity is low. But you
+  should continue to investigate the source of 5XX errors, which could be an
+  external service like Plausible.
```
[the remark this answers is not in the document]
Comment (dhruvkb) [reply documentation/meta/monitoring/runbooks/nuxt_5xx_above_threshold.md:24]: Updated in b763508.

File: documentation/meta/monitoring/runbooks/nuxt_request_count.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  services' usages.
+- If the increase is a botnet attack, we need to block these agents to restore
+  usage to the usual level.
+
+We also need to verify that the requests are being handled properly and that our
+services are capable of meeting this demand (this can be observed from the CPU
+and memory metrics in the ECS dashboards in CloudWatch).
+
+- If our infra can handle the load, there is not much to do except continue to
+  monitor that the resources stay within reasonable limits.
```
[the remark this answers is not in the document]
Comment (dhruvkb) [reply documentation/meta/monitoring/runbooks/nuxt_request_count.md:28]: Updated in 360ac8a.

**Comment by dhruvkb** [comment]
**Update:** the alarms have been merged into `main` and applied in prod so the runbooks should be added to the docs soon.

**Review by krysal — APPROVED** [review]
Excellent 👍

**Review by sarayourfriend — APPROVED** [review]

**Pull request merged**